README.md

```diff
@@ -13,86 +15,97 @@ The [RAPIDS](https://rapids.ai) cuGraph library is a collection of graph analyti
 ```markdown
 import cugraph
 
-# assuming that data has been loaded into a cuDF (using read_csv) Dataframe
+# read data into a cuDF DataFrame using read_csv
 gdf = cudf.read_csv("graph_data.csv", names=["src", "dst"], dtype=["int32", "int32"] )
 
+# We now have data in a COO format (edge pairs)
 # create a Graph using the source (src) and destination (dst) vertex pairs the GDF  
 G = cugraph.Graph()
 G.add_edge_list(gdf, source='src', destination='dst')

```

Comment:
update "add_edge_list" to "from_cudf_edgelist"

Reply:
updated